diagnostic.md

```diff
@@ -7,25 +7,25 @@ Write your answers inside this file, where it's indicated by the comments.
     repository in your answer.
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    git clone git@github.com:MichaelB167/git-basics-diagnostic.git
     ```
 
 2.  What do you do after cloning a repository, but before starting work?
 
-    <!-- Remove this comment and place your answer here. -->
+    initialize git if it's not already with the command git init. then create a working branch with the command git branch <branchName>
```

Comment:
No, if you're cloning you don't need to initialize.  Cloning creates that .git file that initialize does.  All you want to do after cloning is got into the directory, then create a response a branch.